### DISCUSSION TITLE
Anomaly Correlation Coefficient

### DISCUSSION THREAD

**Discussion by cjmelick** [dtcenter/METplus#1878] [Statistical Computation] [answered]
I noticed that Anomaly Correlation Coefficient is computed in Point Stat but was curious if that is also a statistic output in Grid Stat as well.   Thanks!

**Comment by JohnHalleyGotway** [comment, marked as answer]
@cjmelick the anomaly correlation is reported in columns 102 and 120 of the continuous statistic line type (CNT) as seen in [Table 10.7](https://met.readthedocs.io/en/latest/Users_Guide/point-stat.html#id11) of the MET User's Guide.

A more complete definition, including equations, can be found in [Appendix C](https://met.readthedocs.io/en/latest/Users_Guide/appendixC.html#anomaly-correlation-coefficient).

You see that table 10.7 is in the Chapter about Point-Stat and are wondering whether or not Grid-Stat also produces this output. The answer is definitely yes, Grid-Stat does write the CNT output line type, but I do see why that may not be immediately obvious. We of course don't want to repeat info in the Grid-Stat chapter that was already discussed in the preceding Point-Stat chapter. This [grid_stat output section](https://met.readthedocs.io/en/latest/Users_Guide/grid-stat.html#grid-stat-output) does link back to the Point-Stat chapter in the last paragraph.

Big picture, Point-Stat and Grid-Stat write many of the same output line types. Those common ones are documented in the Point-Stat chapter while the ones specific to Grid-Stat are documented in the Grid-Stat chapter.

Hope that helps clarify.

**Reply by cjmelick** [reply]
@JohnHalleyGotway Thanks for the explanation!   That makes sense.   Hope you have a great day!
